tests/languages/django!+css/inclusion.test

```diff
@@ -0,0 +1,43 @@
+<style>
+a {
+	color: {{ bar }};

```

Comment:
Should we test with `{% %}` as the delimiter? I thought this was only for filters.

Reply:
If one works, the other should also work. But sure, better be safe than sorry.